internal/state/statuses.go

```diff
@@ -127,3 +137,16 @@ func buildStatuses(graph *graph) Statuses {
 
 	return statuses
 }
+
+func buildBaseRouteConditions(gcValidAndExist bool) []conditions.RouteCondition {
+	conds := conditions.NewDefaultRouteConditions()
+
+	// FIXME(pleshakov): Figure out appropriate conditions for the cases when:
```

Comment:
Why can't we answer this now?

Reply:
this is to not increase the scope of this PR